docs/Installing.asciidoc

```diff
@@ -311,6 +310,23 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Needle Caching  
+~~~~~~~~~~~~~~~
+
+If you your network is slow or you experience long time to load needles you
```

Comment:
@aaannz, sorry my bad here... you mean the _If you your..._ right? (somehow i skipped this when proofreading :P) - fixed -